packages/botonic-plugin-flow-builder/src/action.tsx

```diff
@@ -8,16 +8,16 @@ import BotonicPluginFlowBuilder from './index'
 
 type FlowBuilderActionProps = {
   content?: FlowContent[]
-  handoffMsg?: HtHandoffNode
+  isHandoff?: boolean
 }
 
 export default class FlowBuilderAction extends React.Component<FlowBuilderActionProps> {
   static contextType = RequestContext
 
   static async botonicInit(request: ActionRequest): Promise<any> {
     const flowBuilderPlugin = request.plugins
-      .flowBuilder as BotonicPluginFlowBuilder
-    const locale = flowBuilderPlugin.getLocale(request.session)
+      .hubtypeFlowBuilder as BotonicPluginFlowBuilder
+    const locale = flowBuilderPlugin.getLocale()
```

Comment:
do not remove session, as services will use it to determine the locale to be used